src/InputManager.cpp

```diff
@@ -166,6 +181,11 @@ void InputManager::process_input(SDL_Event* event) const {
             std::cout << "[Player " << player_id << "] ";
 
             switch (event->caxis.axis) {
+
+                /**
+                 * LEFT STICK
+                 */
```

Comment:
Are such big comments necessary?

`// LEFT STICK`

shoudl suffice right?

Reply:
This would come down to preference. Will follow your preference regardless